assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/bootkube.tf

```
@@ -3,9 +3,9 @@ module "bootkube" {
   source = "../../../bootkube"
 
   cluster_name = var.cluster_name
-  api_servers  = [format("%s.%s", var.cluster_name, var.k8s_domain_name)]
+  api_servers  = [var.k8s_domain_name, format("%s.%s", var.cluster_name, var.k8s_domain_name)]
```

```suggestion
  api_servers  = [format("%s.%s", var.cluster_name, var.k8s_domain_name)]
```
The `k8s_domain_name` is just the suffix similar to the `dns_zone` variable and can't be used alone here. Users need to update their configuration entry as this is a breaking change.